portfolio/src/main/webapp/script.js

```diff
@@ -16,11 +16,39 @@
  * Adds a random greeting to the page.
  */ 
 
-function changeColor() { 
-    let x = Math.floor(Math.random() * 256);
-    let y = Math.floor(Math.random() * 256);
-    let z = Math.floor(Math.random() * 256);
-    let rgb = "rgb(" + x + "," + y + "," + z + ")";
-    document.getElementById('box').style.backgroundColor = rgb;
-    document.getElementById('color').innerHTML = rgb;
+// function changeColor() { 
```

Comment:
Same as other file. Don't leave in commented out code. Either delete (it'll be in git history) or uncomment.